__tests__/app.test.js

```diff
@@ -61,4 +61,57 @@ describe.only('GET api/articles/:article_id', () => {
             expect(body).toEqual({msg: 'Route not found'})
         })
     });
+});
+
+describe('PATCH /api/articles/:article_id', () => {
+    it('should update the vote count by the amount specified', () => {
+        const newVote = { inc_votes: 100 }
+        return request(app).patch('/api/articles/3')
+        .send(newVote)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toEqual({
+                article_id: 3,
+                title: "Eight pug gifs that remind me of mitch",
+                topic: "mitch",
+                author: "icellusedkars",
+                body: "some gifs",
+                created_at: expect.any(String),
+                votes: 100,
+            })
+        })
+    });
+    it('should return a 400 with a bad request if invalid ID is sent', () => {
+        const ID = 'bad'
+        return request(app).patch(`/api/articles/${ID}`)
+        .expect(400)
+        .then(({body}) => {
+            expect(body).toEqual({msg : "Bad Request"})
+        })
+    });
+    it('should return a 400 with a bad request if votes is not a number', () => {
+        const newVote = { inc_votes: 'yes' }
+        return request(app).patch(`/api/articles/3`)
+        .send(newVote)
+        .expect(400)
+        .then(({body}) => {
+            expect(body).toEqual({msg : "Bad Request"})
+        })
+    });
+    it('should return a 404 with Not Found message if no endpoint is given', () => {
+        return request(app).patch(`/api/articles/`)
+        .expect(404)
+        .then(({body}) => {
+            expect(body).toEqual({msg : "Route not found"})
+        })
+    });
+    it('should return a 404 error with /route not found/ when end point is valid type but does not not exist', () => {
+       return request(app).patch('/api/articles/9898989')
+       .expect(404)
+       .then(({body}) => {
+           expect(body).toEqual({msg: "Route not found"})
+       })
+    });
+    // add 404 error handling
```

Comment:
Would like to see more tests. Specifically
- sending a vote object with a broken or missing inc_votes key, such as {} or {votes: 100}